lib/consoletest.pm

```diff
@@ -51,6 +58,31 @@ sub post_fail_hook {
     $self->export_logs_desktop;
 }
 
+=head2 record_avc_selinux_alerts
+
+List AVCs that have been recorded during a runtime of a test module that executes this function
+
+=cut
+
+sub record_avc_selinux_alerts {
+    if ((current_console() !~ /root|log/) || (script_run('test -f /var/log/audit/audit.log') != 0)) {
+        return;
+    }
+
+    my @logged = split(/\n/, script_output('ausearch -m avc -r', proceed_on_failure => 1));
+
+    # no new messages are registered
+    if ($avc_record{end} <= $avc_record{start}) {
```

Comment:
This looks like it's missing some code in between. Or should this be

`if (scalar @logged <= $avc_record{start}) {`

Reply:
damn, forgot to check whether the list `@logged` contains anything reasonable data. Thanks

Reply:
But your suggestion should be enough anyway.

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15975